libexec/rtld-elf/rtld.c

```diff
@@ -1531,15 +1531,20 @@ const char *
 pcc_cap(const Obj_Entry *obj, Elf_Off offset)
 {
 	Elf_Addr addr;
+	const char *pcc;
```

Comment:
Maybe call it `pcc_cap` so that the diff is smaller?

Reply:
Fixed